README.md

```diff
@@ -158,7 +158,7 @@ We have a badge! Link it to your live Storybook example.
 [![Storybook](https://cdn.jsdelivr.net/gh/storybookjs/brand@main/badge/badge-storybook.svg)](link to site)
 ```
 
-If you're looking for material to use in your presentation about storybook, like logo's video material and the colors we use etc, you can find all of that at our [brand repo](https://github.com/storybookjs/brand).
+If you're looking for material to use in your presentation about the storybook, like logo's video material and the colors we use etc, you can find all of that at our [brand repo](https://github.com/storybookjs/brand).
```

Comment:
@sriram-km this is actually almost correct in the original form if you could update this to:

```md
If you're looking for materials to use in your presentation about Storybook, like the logo, video materials, the colors we use, etc, you can find all of that at our [brand repo](https://github.com/storybookjs/brand).
```

Reply:
@jonniebigodes first of all sorry for the late response. Done the change. Please have a look. Thanks